server/resp/src/test/java/org/infinispan/server/resp/RespSetCommandsTest.java

```diff
@@ -0,0 +1,22 @@
+package org.infinispan.server.resp;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.testng.annotations.Test;
+
+import io.lettuce.core.api.sync.RedisCommands;
+
+@Test(groups = "functional", testName = "server.resp.RespSetCommandsTest")
+public class RespSetCommandsTest extends SingleNodeRespBaseTest {
+    @Test
+    public void testSadd() {
+        RedisCommands<String, String> redis = redisConnection.sync();
```

Comment:
we need more tests covering (check lists tests) 
if the key already exist and not a set, should throw an error